### ISSUE TITLE
Metadata TG 2.0 dataset/series: Preactivated Conformance Classes 

### ISSUE THREAD

**Issue by ghost** [INSPIRE-MIF/helpdesk-validator#628]
Hi community,

it seems that preselected Conformance Classes for Metadata TG 2.0 dataset/series on production instance aren't correct.

Status Quo:
![grafik](https://user-images.githubusercontent.com/38787131/136186822-31947fec-6d41-450a-99a8-0a5444929697.png)

In my opinion CC2 shouldn't be preselected.

Could you please deactivate CC2, because this class is mentioned to be only optional!

Kind regards.

**Comment by fabiovin** [comment]
Dear @DeordD,

according to the INSPIRE roadmap, all datasets shall be conformant to IR and this includes also "Metadata for interoperability" since 21/10/2020 (Dark green rhombus in the picture).

For this reason, we decided to include CC2 by default and it will be used for the next Monitoring in December 2021.

![image](https://user-images.githubusercontent.com/46569952/136189450-0fec5994-e734-47f8-bcab-7d9da9938751.png)

**Comment by ghost** [comment]
Hi @fabiovin, 

thanks a lot for the clarification and graphical presentation!

Could you please introduce this change also on ets-repository level too, because ETS-dependency for CC2b isn't correct?

Status Quo:
CC2b is not dependent on CC2, but only on CC1. 

see https://github.com/inspire-eu-validation/ets-repository/blob/4d33556870c6927f6be6b02506639b5639e7c081/metadata/2.0/datasets-and-series/ets-md-monitoring-bsxets.xml#L44

Thanks in advance!

**Comment by fabiovin** [comment]
Dear @DeordD,

thank you for your suggestion.

We have set all CCs by default in the UI to "force" users to check compliance with all requirements.

Since CC2 applies "only" to harmonized datasets, we decided to leave the dependency of the CC2b from the CC1, to allows users to use it also for not harmonized datasets (e.g. datasets out of scope of a theme or not yet harmonised).
